template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPageWithPageModel/.template.config/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page with Page Model",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPageWithPageModel",
+  "shortName": "page",
```

Comment:
I'd advocate for making these into a single template with a distinguishing param, maybe something like:
```
"model": {
  "type": "parameter",
  "datatype": "bool",
  "defaultValue": "false",
  "description": "<insert appropriate desc. here>"
}
}